test/uk/gov/hmrc/gform/services/NumberValidationSpec.scala

```diff
@@ -224,5 +222,4 @@ class NumberValidationSpec extends FlatSpec with Matchers with EitherMatchers wi
     result.toEither should beLeft(Map(fieldValue.id -> Set("number must be at most 2 whole digits and decimal fraction must be at most 1 digits")))
   }
 
-  implicit lazy val hc = HeaderCarrier()
```

Comment:
`hc` was placed at the end of the file intentionally because as we agreed "important goes up". Here test is more important and should be seen first.